app/controllers/api/v1x0/requests_controller.rb

```diff
@@ -26,7 +26,7 @@ def index
                      resource_check('read', params[:request_id], Request) # NotAuthorizedError if current user cannot access parent request
                      Request.find(params[:request_id]).children
                    else
-                     Request.where(:parent_id => nil)
+                     nil
```

Comment:
This would be more of a refactoring, but would be nice if `requests` became a private method then the `index` method calls `requests` with `params`.

The logic could then be written to return early if `request_id` is nil:
```ruby
def requests(params)
  return nil if params[:request_id].blank?

  resource_check('read', params[:request_id], Request) # NotAuthorizedError if current user cannot access parent request
  Request.find(params[:request_id]).children
end
```

Reply:
👍